processor/k8sprocessor/observability/observability_test.go

```diff
@@ -108,6 +113,6 @@ func TestMetrics(t *testing.T) {
 		assert.Equal(t, d.Descriptor.Name, tt.name)
 		require.Len(t, d.TimeSeries, 1)
 		require.Len(t, d.TimeSeries[0].Points, 1)
-		assert.Equal(t, d.TimeSeries[0].Points[0].Value, int64(1))
+		assert.Equal(t, int64(1), d.TimeSeries[0].Points[0].Value, tt.name)
```

Comment:
Why test name here instead of using `t.Run()`?

Reply:
That's a good point, but I should not have touched this line at all. Reverted now.